plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1398,6 +1398,7 @@ blockly.block.dimension_left=If player leaves dimension %1
 blockly.block.item_consumed=If %1 is consumed
 blockly.block.item_damaged=If %3 is damaged, damage between %1 and %2
 blockly.block.item_in_inventory=If player has from %1 to %2 %3 in inventory
+blockly.block.item_enchanted=If item %1 is enchanted with %2 with min level: %3 max level: %4 and min amount of levels spent: %5
```

Comment:
I have checked other enchantments and they don't use :, so omit them so the block matches other blocks:

<img width="891" height="320" alt="image" src="https://github.com/user-attachments/assets/a0a1472e-2561-41ee-9b75-47f583f356c7" />